src/styles/buttons-side.css

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
Have a look at the PR in the upstream repo

Reply:
Done